Jenkinsfile

```diff
@@ -57,6 +57,16 @@ pipeline {
       }
     }
 
+    stage('Apex installation') {
+      steps {
+        // pinned Apex https://github.com/NVIDIA/apex/commit/52e18c894223800cb611682dce27d88050edf1de
+        sh 'git clone https://github.com/NVIDIA/apex.git && \
+            cd apex && \
+            git checkout 52e18c894223800cb611682dce27d88050edf1de && \
+            pip install -v --no-build-isolation --disable-pip-version-check --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" --global-option="--fast_layer_norm" --global-option="--distributed_adam" --global-option="--deprecated_fused_adam" .'
+      }
+    }
```

Comment:
My bad, I just realized we can't do this because the install will take too long for Jenkins runs.

Reply:
Removed.